actions/icon_overlay.rb

```diff
@@ -72,15 +63,15 @@ def self.source_icons_directory
         source_path = ENV['ICON_OVERLAY_SOURCE_PATH']
         if source_path
```

Comment:
This if statement is slightly confusing - are we saying that `ENV['ICON_OVERLAY_SOURCE_PATH']` can sometimes be empty?

If that's the case when you can do something like:
```
source_path = if !ENV['ICON_OVERLAY_SOURCE_PATH'].empty?
                              File.expand_path(source_path)
                         elsif !ENV['ICON_OVERLAY_ASSETS_BUNDLE'].empty?
                               File.expand_path(File.join(ENV['ICON_OVERLAY_ASSETS_BUNDLE'], 'AppIcon.appiconset'))
                         else
                               ENV['ICON_OVERLAY_SOURCE_PATH']
                         end
```

Reply:
And then you can get rid of the `source_path` underneath too

Reply:
So this is essentially a function inside a function?

Reply:
Surely we can just get rid of `source_path` entirely and just do the if/else statement. Each one will return the value right?